### ISSUE TITLE
Console Spam with "Show coordinates in world" option on

### ISSUE THREAD

**Issue by DoTrONoL** [shurik204/better-suggestions#11]
![image](https://github.com/shurik204/better-suggestions/assets/71497422/676a8480-677d-4e6e-b500-89a0c9ee6fb7)

I Insanely get console spam when I try to use /setblock ~ < yeah, with this character and number, etc. Those option makes my console goes bruh, like this:

```
[17:28:11] [Render thread/INFO]: [CHAT] Changed the block at 3010, 6, 1761
[17:28:15] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:15] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:15] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:15] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:23] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:23] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:23] [Render thread/INFO]: [STDOUT]: ~ ~ ~
[17:28:25] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:25] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:26] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:27] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:27] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:28] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:28] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:31] [Render thread/INFO]: [STDOUT]: 3010 3 1763
[17:28:31] [Render thread/INFO]: [STDOUT]: 3010 3 1763
...
```

Minecraft v1.20.2
Fabric Loader 0.15.6
fabric-api-0.91.6+1.20.2

**Comment by DoTrONoL** [comment]
@shurik204 Plans to 1.20.2 ver? I'm still in that. I saw you fixed it for 1.20.4

**Comment by DoTrONoL** [comment]
oh... It does work in 1.20.2, but modrinth only says 1.20.3-1.20.4 compatible
